zerver/tests/test_retention.py

```diff
@@ -21,95 +37,452 @@ class TestRetentionLib(ZulipTestCase):
     def setUp(self):
         # type: () -> None
         super(TestRetentionLib, self).setUp()
-        self.zulip_realm = self._set_realm_message_retention_value('zulip', 30)
-        self.mit_realm = self._set_realm_message_retention_value('mit', 100)
+        self.zulip_realm = self._set_realm_message_retention_value('zulip.com', 30)
+        self.mit_realm = self._set_realm_message_retention_value('mit.edu', 100)
 
     @staticmethod
-    def _set_realm_message_retention_value(realm_str, retention_period):
+    def _set_realm_message_retention_value(domain, retention_period):
         # type: (str, int) -> Realm
-        realm = Realm.objects.get(string_id=realm_str)
+        # Change retention period for certain realm.
+        realm = Realm.objects.filter(domain=domain).first()
         realm.message_retention_days = retention_period
         realm.save()
         return realm
 
     @staticmethod
-    def _change_messages_pub_date(msgs_ids, pub_date):
+    def _change_msgs_pub_date(msgs_ids, pub_date):
         # type: (List[int], datetime) -> Any
-        messages = Message.objects.filter(id__in=msgs_ids).order_by('id')
-        messages.update(pub_date=pub_date)
-        return messages
+        # Update message pud_date value.
+        msgs = Message.objects.filter(id__in=msgs_ids).order_by('id')
+        msgs.update(pub_date=pub_date)
+        return msgs
 
-    def _make_mit_messages(self, message_quantity, pub_date):
+    def _make_mit_msgs(self, msg_qauntity, pub_date):
         # type: (int, datetime) -> Any
-        # send messages from mit.edu realm and change messages pub date
+        # Send messages from mit.edu realm and change messages pub_date.
         sender = UserProfile.objects.filter(email='espuser@mit.edu').first()
         recipient = UserProfile.objects.filter(email='starnine@mit.edu').first()
         msgs_ids = [self.send_message(sender.email, recipient.email, Recipient.PERSONAL) for i in
-                    range(message_quantity)]
-        mit_messages = self._change_messages_pub_date(msgs_ids, pub_date)
-        return mit_messages
+                    range(msg_qauntity)]
+        mit_msgs = self._change_msgs_pub_date(msgs_ids, pub_date)
+        return mit_msgs
 
-    def test_expired_messages_result_type(self):
+    def _send_cross_realm_message(self):
+        # type: () -> int
+        # Send message from bot to users from different realm.
```

Comment:
I think this is singular, based on the function name, so the comment should be:

`# Send message from bot to user in a different realm.`